src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import './UserInfo.scss';
+
+type Props = {
+  name: string;
+  email: string;
+};
```

Comment:
you should reuse User type here
```suggestion
type Props = User;
```